ConditionalConstruct/Implementation/else_if_ladder_using_scanf.c

```diff
@@ -0,0 +1,32 @@
+/*Implementing Else If ladder using scanf*/
+
+#include <stdio.h>
+void main()
+{
+    int number;
+    printf("Enter any number between 1 to 100 and I will tell you in which range it will lie. \n");
+    scanf("%d",&number);
+    printf("\n\n");
+
+    if(number>90)
```

Comment:
Its kind of repeatability. Could you please consider range like 0-25, 26- 50, 51-75 and 76-100. 
This would help others easiliy understand the topic with less effort. 
Am open to your suggestions

Reply:
Yes sir, I can consider the range as you said, but I just thought of making smaller ranges. I will do it.